packages/gn.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Gn < Package
+  description 'GN is a meta-build system that generates build files for Ninja.'
+  homepage 'https://gn.googlesource.com/gn/'
+  version 'dca877f'
+  source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
+  source_sha256 'b58d799b16d20abf92369fe0749c73f7398996f0afa9933517051778a8bb16c3'
```

Comment:
From my experience, this source_sha256 will change everytime you try to download from the chromium repo.  I would suggest a different file.

Reply:
Any idea which one could do the job ? I faced this problem and thus used the `sommelier.rb`'s config

Reply:
Try:
```
  source_url 'file:///dev/null'
  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
```